ROADMAP.md

```diff
@@ -43,42 +55,52 @@ Future versions of the spec may include these concepts:
 
 - Caching
 - Batching
-- Versioning mechanism for servers/clients to communicate what versions they support
-- Modularity - A way to communicate what features (and possibly versions) of the HTTP spec are supported by a server
+- Versioning mechanism for servers/clients to communicate what versions they
+  support
+- Modularity - A way to communicate what features (and possibly versions) of the
+  HTTP spec are supported by a server
 - Persisted queries
 - Multipart requests (file uploads)
 - Submit MIME type application/graphql+json to IANA
-- New HTTP SEARCH method and how it could be used https://tools.ietf.org/html/draft-snell-search-method-01
+- New HTTP SEARCH method and how it could be used
+  https://tools.ietf.org/html/draft-snell-search-method-01
 
 ## Stages
 
-The process of writing this specification may proceed according this rough outline of stages.
-We are currently in the *Preliminary Stage*.
+The process of writing this specification may proceed according this rough
+outline of stages. We are currently in the _Preliminary Stage_.
 
 ### Stage 0: Preliminary
 
-In the *Preliminary Stage*, things may change rapidly and no one should count on any particular
-detail remaining the same.
+In the _Preliminary Stage_, things may change rapidly and no one should count on
+any particular detail remaining the same.
 
-- If a PR has no requests for changes for 2 weeks then it should be merged by one of the maintainers
-- If anyone has an objection later, they just open a PR to make the change and it goes through the same process
+- If a PR has no requests for changes for 2 weeks then it should be merged by
```

Comment:
Is the thinking here to have some pressure for people to at least formulate some objections?

Reply:
The goal was to keep the process moving.  At this stage, we did not want a PR to be sitting for too long.